MetaMask/metamask-extension
# UX: Autofocus the Address field in Import NFT modal

## Explanation

Both the `Address` and `Token ID` inputs errantly have the `autoFocus` attributes.  The Address should have autoFocus, so this removes the duplicate property.

## Manual Testing Steps

1.  From the home screen, click the `NFTs` tab
2. Click "Import NFT"
3. See "Address" field immediately focus

## Pre-merge author checklist


## Pre-merge reviewer checklist


If further QA is required (e.g. new feature, complex testing steps, large refactor), add the `Extension QA Board` label.

In this case, a QA Engineer approval will be be required.
